### PR TITLE
adding snap installtion command

### PR DESCRIPTION
**Pull request by purveshpatel511**
Released this package as snap package. adding snap installtion method in readme and added it into chagelog files.

### CODE REVIEW

**Comment by sharkdp** [comment]
Thank you for creating a package.

We have package/executable names "bat" and "batcat" already. Would it be possible to prevent yet another new name? Is `bat` already taken?

**Comment by purveshpatel511** [comment]
In snapstore, `bat` is not available for register. And thanks for pointing out the name `batcat`. I register snap with that name and package have been released on snapstore.

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,10 @@
 
 - Running `bat` without arguments fails ("output file is also an input"), see #1396
 
+## Packaging
+
+- `bat` is now available on snapstore with package name called `batcat`, see #1401 (@purveshpatel511)
+
```
Comment (sharkdp) [inline CHANGELOG.md:27]: Could you please move that up to the "unreleased" section? Right now, it's in 0.17.1, which has already been released

Comment (purveshpatel511) [reply CHANGELOG.md]: @sharkdp sure.

**Comment by purveshpatel511** [comment]
Packaging moved to "unreleased" section.

**Pull request merged**